tf_encrypted/protocol/pond.py

```diff
@@ -2781,6 +2787,85 @@ def _avgpool2d_masked(
         return PondPrivateTensor(prot, y_on_0, y_on_1, x.is_scaled) * scalar
 
 
+#
+# batch_to_space_nd and space_to_batch_nd helpers
+#
+
+
+def _batch_to_space_nd_core(prot, tensor, block_shape, crops):
+    x_on_0, x_on_1 = tensor.unwrapped
```

Comment:
maybe rename `tensor` to `x` to match with `x_on_0` etc

Reply:
applies to `_space_to_batch_nd_core` as well

Reply:
would probably rather go the other way -- `tensor` is more informative than `x`

Reply:
jup, either works!